src/Microsoft.SqlTools.Serialization/Microsoft.SqlTools.Serialization.csproj

```diff
@@ -15,8 +15,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.SqlTools.Credentials\Microsoft.SqlTools.Credentials.csproj" />
+    <ProjectReference Include="..\Microsoft.SqlTools.Contracts\Microsoft.SqlTools.Contracts.csproj" />
     <ProjectReference Include="..\Microsoft.SqlTools.Hosting\Microsoft.SqlTools.Hosting.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Contracts\" />
+  </ItemGroup>
```

Comment:
Why is this folder include necessary? Isn't everything you need included by the ProjectReference above?